app/react-native/src/preview/components/OnDeviceUI/index.tsx

```diff
@@ -136,13 +139,12 @@ export default class OnDeviceUI extends PureComponent<OnDeviceUIProps, OnDeviceU
         >
           <Animated.View style={previewWrapperStyles}>
             <Animated.View style={previewStyles}>
-              <Preview
-                accessible={false}
-                disabled={tabOpen === PREVIEW}
-                onPress={this.handleOpenPreview}
-              >
+              <Preview disabled={tabOpen === PREVIEW}>
                 <StoryView url={url} onDevice stories={stories} />
               </Preview>
+              {tabOpen !== PREVIEW ? (
```

Comment:
@Gongreg why do we need this AND the disabled prop above?

Reply:
For styling purposes.